src/main/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceService.scala

```diff
@@ -86,6 +90,14 @@ trait WorkspaceService extends HttpService with PerRequestCreator with FireCloud
             }
           }
         } ~
+        path("updateAttributes") {
+          patch {
+            entity(as[Seq[AttributeUpdateOperation]]) { attributeOp =>
```

Comment:
I am reconsidering my initial approval of this. I think we should just pass whatever entity is sent through as the entity without coercing it to `[Seq[AttributeUpdateOperation]]`. This will save you a bunch of code and simplify the passthrough. You can be more clear about what `AttributeUpdateOperation` is in the yaml, but I don't think we need this in the code.